tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -214,6 +217,87 @@ def test_fsx_lustre_backup(region, pcluster_config_reader, clusters_factory, os,
     _test_delete_manual_backup(remote_command_executor, manual_backup, region)
 
 
+@pytest.mark.dimensions("us-west-2", "c5.xlarge", "alinux2", "slurm")
+@pytest.mark.usefixtures("instance")
+def test_existing_fsx(
+    region, existing_fsx, pcluster_config_reader, clusters_factory, os, scheduler, vpc_stack, test_datadir
+):
+    """
+    Test existing Fsx file system
+
+    Check fsx_fs_id provided in config file can be mounted correctly
+    """
+    mount_dir = "/fsx_mount_dir"
+    bucket_name = existing_fsx[1]
+    cluster_config = pcluster_config_reader(
+        bucket_name=bucket_name, mount_dir=mount_dir, fsx_fs_id=existing_fsx[0].cfn_resources["existingfsx"]
+    )
+    cluster = clusters_factory(cluster_config)
+
+    _test_fsx_lustre(cluster, region, scheduler, os, mount_dir, bucket_name, storage_type=None, auto_import_policy=None)
+
+
+@pytest.fixture()
+def existing_fsx(vpc_stack, cfn_stacks_factory, request, region, key_name, s3_bucket_factory, test_datadir):
```

Comment:
let's use a more generic name for the feature, like `fsx_creator`, `fsx_template` or something like this.

Reply:
Done